pom.xml

```diff
@@ -45,7 +45,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <scala.version>2.12.8</scala.version>
         <scala.compat.version>2.12</scala.compat.version>
-        <geotools.version>17.0</geotools.version>
+        <geotools.version>23.2</geotools.version>
+        <jts.version>1.16.1</jts.version>
```

Comment:
NB:  JTS 1.17.1 is available.:)  

That said, using the version from GeoTools is probably wise!